### ISSUE TITLE
Make creating a Color more consistent?

### ISSUE THREAD

**Issue by icefoxen** [ggez/ggez#210]
Right now there's two ways to make a `Color`: `Color::new(f32, f32, f32, f32)` and `Color::from(various u8 types)`

```
14:15 < Icefoz> Aaronepower: The thing is `Color::new` takes 4 f32's, while there are various `From` implementations for different types.
14:15 < Icefoz> I could be talked into having no `Color::new` and just various `From` implementations, I suppose.
14:16 < Icefoz> Aaronepower: Internally, a Color is 4 f32's.  So `new()` does no conversions, and `from()` does conversions.
14:18 < Ralith> Aaronepower: you expected Color::new to be the exact same as various different From impls?
14:18 < Aaronepower> Icefoz: I know, I'm saying that's unintuitive from a user standpoint. At no point until I looked at the source code did I realise that the two were different.
14:19 < Icefoz> Aaronepower: That's valid, I'll open an issue for it.
14:19 < Ralith> there are more than two
14:19 < Ralith> imo Icefoz's API design here makes perfect sense
14:19 < Ralith> the native, lossless, conversion-free representation *should* be privileged
```

**Comment by icefoxen** [comment]
What we REALLY need is better documentation (done) and `from_rgb(u32)` and `from_rgba(u32)` methods on `Color`.

These changes, along with #209 should be backported into 0.3

**Comment by icefoxen** [comment]
Added to/from u32 types but they really seem less wossname than to/from u8 types so maybe those should be canonical rather than the From impl idk

**Comment by icefoxen** [comment]
Resolved in 2db5edaaa825c41e0f0a99323a571197c729d366
